trade-finance-manager-ui/templates/case/case.njk

```diff
@@ -1,19 +1,33 @@
 {% from "moj/components/sub-navigation/macro.njk" import mojSubNavigation %}
 {% import './_macros/case-summary.njk' as caseSummary %}
+{% import './_macros/case-summary-ndb.njk' as caseSummaryNdb %}
 
 {% extends "index.njk" %}
 {% set primary = 'cases' %}
 
 {% block content %}
 
-  {% set componentData = {
-    deal: deal,
-    tfm: tfm,
-    user: user,
-    amendment: amendment
-  } %}
+  {% if tfm.nonDelegatedBank %}
+    {% set componentData = {
+      deal: deal,
+      tfm: tfm,
+      user: user,
+      amendment: amendment
+    } %}
 
-  {{caseSummary.render(componentData)}}
+    {{caseSummaryNdb.render(componentData)}}
+
+  {% else %}
+    {% set componentData = {
+      deal: deal,
+      tfm: tfm,
+      user: user,
+      amendment: amendment
+    } %}
```

Comment:
Could this `set componentData` outside of the conditional? DRY

Reply:
Your request has been fulfilled :P Just waiting for any @abhi-markan comments and then will push this change